packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -14,11 +14,1064 @@ The Okta Administrator Roles API provides operations to manage administrative ro
+| roleIdOrName   | id or label of the role              | URL          | String                                | TRUE     |
+| permissionType | permission to add to the role        | URL          | [Permission type](#permission-types)) | TRUE     |
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3/permissions/okta.users.manage"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+### Delete permission
+<ApiOperation method="delete" url="/api/v1/iam/roles/${roleIdOrName}/permissions/${permissionType}" />
+
+Deletes a permission from an existing role
+
+#### Request parameters
+
+| Parameter      | Description                          | Param Type   | DataType                              | Required |
+| :------------- | :----------------------------------- | :----------- | :------------------------------------ | :------- |
+| roleIdOrName   | id or label of the role              | URL          | String                                | TRUE     |
+| permissionType | permission to remove from the role   | URL          | [Permission type](#permission-types)) | TRUE     |
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles/cr0Yq6IJxGIr0ouum0g3/permissions/okta.users.manage"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+### Delete role
+<ApiOperation method="delete" url="/api/v1/iam/roles/${roleIdOrName}" />
+
+Deletes a custom role
+
+#### Request parameters
+
+| Parameter      | Description                          | Param Type   | DataType                              | Required |
+| :------------- | :----------------------------------- | :----------- | :------------------------------------ | :------- |
+| roleIdOrName   | id or label of the role              | URL          | String                                | TRUE     |
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/roles/UserCreator"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+## Resource-set operations
+<ApiLifecycle access="beta" />
+These operations allow the creation and manipulation of resource sets as custom collections of resources. Resource-sets are used to assign [custom roles](#custom-role-operations) to administrators.
```

Comment:
".. to administrators, scoped to the designated resources."

Reply:
added as suggested